roles/wp-cli/defaults/main.yml

```diff
@@ -1,2 +1,4 @@
 phar_url: https://raw.githubusercontent.com/wp-cli/builds/gh-pages/phar/wp-cli.phar
 bin_path: /usr/bin/wp
+wp_cli_completion_url: https://raw.githubusercontent.com/wp-cli/wp-cli/master/utils/wp-completion.bash
```

Comment:
Agreed. That's why I splitted it into 2 commits. I don't know why #392 is not merged yet though?